src/main/java/com/webank/webase/node/mgr/front/FrontService.java

```diff
@@ -312,4 +320,38 @@ public void updateFrontWithInternal(Integer frontId, Integer status) {
         updateFront.setStatus(status);
         frontMapper.update(updateFront);
     }
+
+    @Transactional(propagation = Propagation.REQUIRED)
+    public TbFront insert(String nodeId,
```

Comment:
different from newFront method, not request this Front until start successfully.